### ISSUE TITLE
Add name-of-game column to Markdown tables

### ISSUE THREAD

**Issue by jp-bennett** [HonkingGoose/proton_codecs_media_support_tracker#20]
The tables are currently unsearchable unless you already know the upstream ticket number. It seems like this really cuts down on the usefulness of having this data at all. As a user, looking to add a game report, it's exceptionally difficult to determine if a given game is already covered. Adding a [name of game] column seems like the way to go.

**Comment by HonkingGoose** [comment]
Thanks for your feedback. I agree that having the name in the table is easier.

The main problem is that the tables are getting too wide to display comfortably as-is, and adding a _name of game_ column will make the tables even wider. I'm keeping the Markdown tables as they are now.

Having said that, I think the "real fix" is something like this:

1. Migrate the Markdown tables to a proper data format like JSON.
1. Add `steamID` property/field to JSON/YAML.
1. Have somebody else build a static website that displays the JSON data in a nice table that's human readable/searchable/responsive. They can also figure out how to use the steamID to grab data from the Steam Store webpages.
1. On the external website there's a link back to this GitHub repository to add a new table entry with a pull request.

In effect doing the above would make this project an API that somebody else can then extend from. This would also make my life easier for this project, as somebody else can then use their whizzbang Vue/Angular/React/Rails skills to display the JSON in a nice table that's hosted on a static website. The Proton developers can then use that website as a front-end to the data hosted on this repository.

I've already discussed this "real fix" at #13, and asked Proton developer **aeikum** what he needs from this project.
The proton developers are happy with the current plain-text Markdown tables, and don't have a need for another data format.

I myself am not able to figure out what the JSON data format should look like, so I'm blocked from migrating. Also if I migrate, I'm then dependent on somebody else to care enough (very unlikely) to make a simple webpage that displays the tables in a usable manner, as I myself lack the necessary know-how to make and maintain such a website.

So in short, unless somebody kind comes along and helps/mentors me with both the JSON file, and with making a real basic website that I can host for free on GitHub pages, this feature will not get implemented.
